### ISSUE TITLE
Question regarding loss function

### ISSUE THREAD

**Issue by Peachypie98** [DAMO-DI-ML/KDD2023-DCdetector#39]
I have a question,

The final loss function is N - P which is Patch Wise Loss - In-Patch Loss and the code shows loss = prior_loss - series_loss(meaning, prior: Patch Wise, series: In-Patch). But after debugging line by line, i think it should be the other way around, which is loss = series_loss - prior_loss because the prior_loss shows In-Patch loss. Or it doesn't really matter since those two losses are identical in terms of value?

Also, does this model can identify anomaly across different dimensions? (like in which dimension and timepoint the anomaly is detected). For now, it seems the model can only do time point anomaly detection.

**Comment by yyysjz1997** [comment]
> I have a question,
> 
> The final loss function is N - P which is Patch Wise Loss - In-Patch Loss and the code shows loss = prior_loss - series_loss(meaning, prior: Patch Wise, series: In-Patch). But after debugging line by line, i think it should be the other way around, which is loss = series_loss - prior_loss because the prior_loss shows In-Patch loss. Or it doesn't really matter since those two losses are identical in terms of value?
> 
> Also, does this model can identify anomaly across different dimensions? (like in which dimension and timepoint the anomaly is detected). For now, it seems the model can only do time point anomaly detection.

Hi. We have updated the descriptions in the [second version paper](https://arxiv.org/abs/2306.10347). FYI. Besides, our method can solve the time series anomaly detection problem in single and multi-dimensional time series, which cannot determine the anomalies of certain channels because of the fusion of multi-channel features. However, easy improvements can be made to achieve the requirements you mentioned, such as iterating or designing new anomaly scores. You can try it in our public code. Good Luck!
